Error: Cannot find module 'dispensary'

When I try to run `addons-linter`, it says:
```
Error: Cannot find module 'dispensary'
    at Function.Module._resolveFilename (module.js:472:15)
    at Function.Module._load (module.js:420:25)
    at Module.require (module.js:500:17)
    at require (internal/module.js:20:19)
    at Object.module.exports.Object.defineProperty.value (/Users/lcrouch/code/shield-studies-addon-template/node_modules/addons-linter/dist/webpack:/external "dispensary":1:1)
    at __webpack_require__ (/Users/lcrouch/code/shield-studies-addon-template/node_modules/addons-linter/dist/webpack:/webpack/bootstrap 4ebb057115de6d1d7f94:19:1)
    at Object.<anonymous> (/Users/lcrouch/code/shield-studies-addon-template/node_modules/addons-linter/dist/webpack:/src/linter.js:5:1)
    at __webpack_require__ (/Users/lcrouch/code/shield-studies-addon-template/node_modules/addons-linter/dist/webpack:/webpack/bootstrap 4ebb057115de6d1d7f94:19:1)
    at Object.<anonymous> (/Users/lcrouch/code/shield-studies-addon-template/node_modules/addons-linter/dist/webpack:/src/main.js:2:1)
    at Object.module.exports.module.exports.module.deprecate (/Users/lcrouch/code/shield-studies-addon-template/node_modules/addons-linter/dist/addons-linter.js:94:31)
```

It looks like the dispensary module does is missing the `dist/dispensary.js` file indicated as the main module by `package.json`.

I think it's because `dist/*` is included in the `.gitignore` file, and there is no `.npmignore` file, so the `dist/` directory is left out of the npm package completely.

--- a/.npmignore
+++ b/.npmignore
@@ -1,0 +1,14 @@
+.babelrc
+.eslintrc
+.jscsrc
+.npmignore
+.nvmrc
+.travis.yml
+CONTRIBUTING.md
+Gruntfile.js
+coverage
+node_modules
+src
+tasks
+tests
+webpack.config.js

--- a/package.json
+++ b/package.json
@@ -13,7 +13,8 @@
     "update": "npm run build && ./bin/dispensary > src/hashes.txt",
     "start": "node -e \"require('grunt').cli()\" null start",
     "build": "node -e \"require('grunt').cli()\" null build",
-    "test": "LANG='en_US.UTF-8' node -e \"require('grunt').cli()\" null test"
+    "test": "LANG='en_US.UTF-8' node -e \"require('grunt').cli()\" null test",
+    "prepublish": "npm test && npm run build"
   },
   "repository": {
     "type": "git",

